src/objects/Mesh.js

```diff
@@ -178,13 +178,13 @@ Mesh.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		}
 
-		function checkBufferGeometryIntersection( object, raycaster, ray, position, uv, a, b, c ) {
+		function checkBufferGeometryIntersection( object, material, raycaster, ray, position, uv, a, b, c ) {
```

Comment:
Oh gosh. `ray` is in a different space?

Also, I see object, material, raycaster passed through to the lowest-level methods. Maybe you can refactor at some point... if you want...